lapis/drone.py

```diff
@@ -11,20 +11,24 @@ class ResourcesExceeded(Exception):
 
 
 class Drone(interfaces.Pool):
-    def __init__(self, scheduler, pool_resources: dict, scheduling_duration: float, exclusive: bool=False,
-                 ignore_resources: list=None):
+    def __init__(self, scheduler, pool_resources: dict,
+                 scheduling_duration: float, exclusive: bool = False,
+                 ignore_resources: list = None):
         """
         :param scheduler:
         :param pool_resources:
         :param scheduling_duration:
-        :param exclusive: Determines if the drone is used exclusively by jobs in sequential order
+        :param exclusive: Determines if the drone is used exclusively by jobs
+        in sequential order
         """
         super(Drone, self).__init__()
         self.scheduler = scheduler
         self.pool_resources = pool_resources
         self.resources = {resource: 0 for resource in self.pool_resources}
         if ignore_resources:
-            self._valid_resource_keys = [resource for resource in self.pool_resources if resource not in ignore_resources]
+            self._valid_resource_keys = [
+                resource for resource in self.pool_resources
+                if resource not in ignore_resources]
```

Comment:
The next (dict) comprehension has a closing mark on the next line, this (list) comprehension has a closing mark trailing. I recommend using one style for comprehensions; I prefer using an extra line for the closing mark.

Reply:
I added another commit to fix this and also put the closing mark on a new line.